osquery/tables/applications/browser_chrome.cpp

```diff
@@ -11,6 +11,12 @@
 
 namespace fs = boost::filesystem;
 
+namespace {
+fs::path chrome_path;
+fs::path chromium_path;
+fs::path brave_path;
+} // namespace
+
```

Comment:
Remove this, and instead declare these variables in the `genChromeExtensions` method. Generally prefer to declare variables as close to where they are first used. This mainly helps with readability but also helps protect against bugs by minimizing the their scope.

For example: place `fs::path chrome_path` between lines 29 and 30. And similarly `fs::path brave_path;` right before line 39, etc.